_dashboards/management/connect-prometheus.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title: Connecting Prometheus to OpenSearch
+parent: Data sources
+nav_order: 20
+---
+
+# Connecting Prometheus to OpenSearch
+
```

Comment:
@AntonEliatra Please let me know if you cut this content from my previously published content. It appears to be a cut and paste from https://opensearch.org/docs/latest/dashboards/management/multi-data-sources/#step-2-create-a-new-data-source-connection. Let me know which text actually was added or edited.

Reply:
Hi @vagimeli, I'm not sure which content you are referring to, My commits are available in this PR, the very first one is https://github.com/opensearch-project/documentation-website/pull/7800/commits/9a13153fcd0c66154fd7a12e336f55410f05bb8b

A lot of the parts were moved around

Reply:
@AntonEliatra I'll take over this PR as the tech writer to and work with the feature's developer to address information gaps. Thanks for writing the initial content.